app/cli/.gitignore

```diff
@@ -0,0 +1,3 @@
+badgeyay.egg-info/
```

Comment:
Can't we include this in the gitignore at the root?

Reply:
Actually this library will be placed  in root folder. As i will complete the code, will surely put the library in the root folder, or if possible in a different branch or repo? May be?